test/Rings/groebner-test.jl

```diff
@@ -1,37 +1,39 @@
-@testset "groebner" begin
-    R, (x, y) = PolynomialRing(QQ, ["x", "y"])
-    I = ideal(R,[x*y^2 - x, x^3 - 2*y^5])
-    @test leading_ideal(I) == ideal(R,[x*y^2, x^4, y^5])
-    @test leading_ideal(I, :lex) == ideal(R,[y^7, x*y^2, x^3])
-end
-
-@testset "groebner leading ideal" begin
-   R, (t, x, y, z) = PolynomialRing(QQ, ["t", "x", "y", "z"])
+#@testset "groebner" begin
```

Comment:
Please add a comment explaining why this is disabled.

Also, perhaps use a multiline comment (`#= ... =#`) for a smaller diff?

Basically, code should either be deleted; or if it is retained and for some reason just commented out, this should be explained in a comment.

Reply:
This is outdated anyway; I'm waiting for #1161 .
But I will do so in the future.